projects/tesseract-ocr.github.io/package.yml

```diff
@@ -0,0 +1,48 @@
+distributable:
+   url: https://github.com/tesseract-ocr/tesseract/archive/refs/tags/{{version}}.tar.gz
+   strip-components: 1
+
+versions:
+  github: tesseract-ocr/tesseract/tags
+
+dependencies:
+  cairographics.org: 1
+  unicode.org: 71
+  leptonica.org: '*'
+  libarchive.org: '*'
+  gnome.org/pango: 1
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    gnu.org/autoconf: '*'
+    gnu.org/automake: '*'
+    gnu.org/libtool: '*'
+    freedesktop.org/pkg-config: '*'
+    gnu.org/wget: '*'
+  script: |
+    ./autogen.sh
+    ./configure $ARGS
+    make --jobs {{ hw.concurrency }} 
+    make install 
+
+    # install trained datafiles and move to share/tessdata
+    wget https://github.com/tesseract-ocr/tessdata/blob/main/eng.traineddata?raw=true -O eng.traineddata
+    wget https://github.com/tesseract-ocr/tessdata/blob/main/osd.traineddata?raw=true -O osd.traineddata
+    mv eng.traineddata "{{prefix}}"/share/tessdata/
+    mv osd.traineddata "{{prefix}}"/share/tessdata/
+    
+  env: 
+    ARGS: 
+      - --prefix="{{prefix}}"
+      - --disable-dependency-tracking
+      - --datarootdir="{{prefix}}"/share
+
+provides:
+  - bin/tesseract
+
+test:
+  script: |
+    export TESSDATA_PREFIX="{{prefix}}"/share/tessdata/
```

Comment:
should this be a top-level `runtime.env.TESSDATA_PREFIX` key?

Reply:
If it's required for the test then it will indeed be required at runtime.

Reply:
@jhheider could you help me understand what you mean by `runtime.env`?

do I add an `env` key in the first level of the package.yml? I couldn't find a package that has a similar setup 

```sh
...
dependencies:
    - ...

env:
    - TESSDATA_PREFIX="{{prefix}}"/share/tessdata/

build:
...
```

I tried that locally but adding `echo $TESSDATA_PREFIX` to test and running `pkg test` is empty